src/app/treasure-hunt/treasure-hunt-report/executive-summary/carbon-emissions-summary-table/carbon-emissions-summary-table.component.html

```diff
@@ -0,0 +1,144 @@
+<label class="group-label">CO<sub>2</sub> Emission Results</label>
+<table #copyTable class="table table-striped table-hover">
+    <tbody>
+        <tr class="bold">
+            <td>
+                Utility
+            </td>
+            <td>
+                Current CO<sub>2</sub> Emissions (tonne CO<sub>2</sub>)
+            </td>
+            <td>
+                Projected CO<sub>2</sub> Emissions (tonne CO<sub>2</sub>)
+            </td>
+            <td>
+                CO<sub>2</sub> Emission Savings (tonne CO<sub>2</sub>)
+            </td>
+        </tr>
+        <!--Electricity-->
+        <tr *ngIf="energyUsage.electricityUsed">
+            <td class="bold">
+                Electricity
+            </td>
+            <td>
+                {{carbonResults.electricityCO2CurrentUse | number:'1.0-0'}}
+            </td>
+            <td>
+                {{carbonResults.electricityCO2ProjectedUse | number:'1.0-0'}}
+            </td>
+            <td>
+                {{carbonResults.electricityCO2CurrentUse - carbonResults.electricityCO2ProjectedUse | number:'1.0-0'}}
```

Comment:
Why not add a savings property to the interface for each result? Doing this calculation in html isn't ideal